transbank/onepay/transaction.py

```diff
@@ -0,0 +1,122 @@
+# encoding: utf-8
+import calendar
+import http.client
+import json
+
+from datetime import datetime
+from enum import Enum
+from urllib.parse import urlparse
+
+from transbank.onepay.schema import ItemSchema, TransactionCreateRequestSchema, TransactionCreateResponseSchema, SendTransactionResponseSchema
+
+from transbank.onepay.cart import ShoppingCart
+from transbank.onepay.error import TransactionCreateError, SignError
+from transbank.onepay import sign
+
+from transbank import onepay
+
+class Channel(Enum):
+    WEB = "WEB"
+    MOBILE = "MOBILE"
+    APP = "APP"
+
+class Options(object):
+    def __init__(self, api_key: str, shared_secret: str):
+        self.api_key = api_key
+        self.shared_secret = shared_secret
+
+class TransbankRequestResponse(object):
+    signable_attributes = []
+
+    def get_signable_data(self, append_data = []):
```

Comment:
I'd rename this method to `signable_data` (no need for the `get` prefix, I think) and remove the optional `append_data` argument (you'll see why in a moment).